packages/backend/package.json

```diff
@@ -21,7 +21,8 @@
     "compression": "^1.7.4",
     "cors": "^2.8.5",
     "express": "^4.17.1",
-    "helmet": "^3.22.0"
+    "helmet": "^3.22.0",
+    "winston": "^3.2.1"
```

Comment:
Should be a dev dep

Reply:
yeah now it can be, couldn't in the vm2 experiment - good catch